modules/@apostrophecms/login/index.js

```diff
@@ -196,6 +197,20 @@ module.exports = {
             return user;
           }
         } : {})
+      },
+      get: {
+        async context () {
+          try {
+            const aposPackage = await fs.readJson('./node_modules/apostrophe/package.json');
```

Comment:
@boutell Stuart wanted the Apostrophe version, not the project one. Is this correct or should I use `self.apos.rootDir`?